roles/patchman_client/tasks/update.yml

```diff
@@ -1,8 +1,11 @@
 ---
 - name: Wait for patchman service
-  ansible.builtin.wait_for:
-    host: "{{ patchman_client_host }}"
-    port: "{{ patchman_client_port }}"
+  ansible.builtin.uri:
+    url: "{{ patchman_client_server_url }}"
```

Comment:
Do you also want to change the definition of patchman_client_server_url to be depending on patchman_traefik or do you expect deployers to override it accordingly?

Reply:
Not sure. The deployer should take care of it for the moment.